locale/pt_BR/electrum.po

```diff
@@ -477,7 +477,7 @@ msgstr "Desencriptar"
 
 #: gui/qt/network_dialog.py:217
 msgid "Default Servers"
-msgstr "Servidores Padrão"
+msgstr "Servidor Padrão"
```

Comment:
Shouldn't this be the plural form?

Reply:
Default ( exacly? ) 
or 
Defaults ( read again )

Reply:
Pardon?
What i mean is that "Servidor" == "Server" and not "Servers" (plural) as it should.
So i think the sentence "Servidores Padrão" is correct. I'm not a native speaker so please correct me if i'm wrong.